OpenKh.Tools.BbsMapStudio/OpenKh.Tools.BbsMapStudio.csproj

```diff
@@ -0,0 +1,62 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<OutputType>WinExe</OutputType>
+		<TargetFramework>net5.0-windows</TargetFramework>
+        <LangVersion>latest</LangVersion>
+		<AssemblyTitle>BBS Map Studio</AssemblyTitle>
+		<Product>BBS Map Studio - OpenKh</Product>
+		<AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+		<ApplicationIcon />
+		<StartupObject />
+	</PropertyGroup>
+
+	<ItemGroup>
+		<MonoGameContentReference Include="**\*.mgcb" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<Compile Remove="Content\bin\**" />
+		<EmbeddedResource Remove="Content\bin\**" />
+		<MonoGameContentReference Remove="Content\bin\**" />
+		<None Remove="Content\bin\**" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<PackageReference Include="ImGui.NET" Version="1.78.0" />
+		<PackageReference Include="MonoGame.Content.Builder" Version="3.7.0.9" />
+		<PackageReference Include="MonoGame.Framework.DesktopGL.Core" Version="3.7.0.7" />
+		<PackageReference Include="AssimpNet" Version="5.0.0-beta1" />
+		<PackageReference Include="McMaster.Extensions.CommandLineUtils" Version="3.1.0" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<ProjectReference Include="..\OpenKh.Engine.MonoGame\OpenKh.Engine.MonoGame.csproj" />
+		<ProjectReference Include="..\OpenKh.Engine\OpenKh.Engine.csproj" />
+		<ProjectReference Include="..\OpenKh.Kh2\OpenKh.Kh2.csproj" />
```

Comment:
Do you have an import feature for KH2 stuff or did you forget to remove reference?

Reply:
I'll make sure to remove it once I fixed all the still in use references.